src/utils.js

```diff
@@ -41,8 +41,8 @@ let trimPath = (path) => {
     return path?.replace(/^\//, '');
 }
 
-let clearCache = () => {
-    fs.removeSync('temp');
+let clearCache = (cacheDir) => {
+    fs.rmSync(cacheDir,{recursive:true});
```

Comment:
Can use `rmdirSync` right ? Instead of passing another param ? Check [here](https://www.geeksforgeeks.org/node-js-fs-rmdirsync-method/)